go/noise/noise.go

```diff
@@ -100,4 +118,14 @@ type Noise interface {
 	// satisfies (epsilon,deltaNoise+deltaThreshold)-differential privacy under the
 	// given assumptions of L_0 and L_∞ sensitivities.
 	Threshold(l0Sensitivity int64, lInfSensitivity, epsilon, deltaNoise, deltaThreshold float64) float64
+
+	// ReturnConfidenceIntervalInt64 will return a ConfidenceIntervalInt64 structure using the int64 noisedValue,
```

Comment:
"struct" instead of "structure".

Reply:
Also it might be easier to parse if we move "with the confidenceLevel given" after "ConfidenceIntervalInt64 struct", so it would be:

"ReturnConfidenceIntervalInt64 will return a ConfidenceIntervalInt64 struct with the given confidenceLevel using the..."

(Also below)